client/src/components/podcasts/PodcastReviewForm.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+
+const ratings = ['', '★', '★ ★', '★ ★ ★', '★ ★ ★ ★', '★ ★ ★ ★ ★']
+
+const PodcastReview = ({ postReview }) => {
+
+  const [podcastReviewRecord, setPodcastReviewRecord] = useState({
+    description: "",
+    rating: ""
+  })
+
+  const ratingOptions = ratings.map(rating => {
+    return (
+      <option key={rating} value={rating}>
+        {rating}
+      </option>
+    )
+  })
+
+  const handleInputChange = event => {
+    setPodcastReviewRecord({
+      ...podcastReviewRecord,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    postReview(podcastReviewRecord)
+    clearForm()
+  }
+
+  const clearForm = event => {
+    setPodcastReviewRecord({
+      description: "",
+      rating: ""
+    })
+  }
+ 
+  return (
+    <div className="callout">
+      <h1>Write a review</h1>
+      <form onSubmit={handleSubmit}>
+        <label>
+          <input
```

Comment:
Yes we could omit those properties label `htmlFor` and input `id` because the input is nested inside the label, however just be mindful in the future you may need to add them if you were to test this component